templates/vault_main_configuration.hcl.j2

```diff
@@ -87,6 +87,9 @@ listener "tcp" {
   {% include vault_backend_raft with context -%}
 {% endif %}
 
+{% if vault_disable_mlock -%}
```

Comment:
I'd expect `if vault_disable_mlock is defined`

Reply:
Oh dear I believe you're right.

Reply:
I just submitted a correction.